contracts/BountyRegistry.sol

```diff
@@ -554,15 +562,14 @@ contract BountyRegistry is Pausable {
 
                         if (malicious == consensus) {
                             expertRewards[j] = expertRewards[j].add(assertions[j].bid.mul(ap.loserPool).div(ap.winnerPool));
-                            expertRewards[j] = expertRewards[j].add(bounty.amount.mul(ap.loserPool).div(ap.winnerPool));
+                            expertRewards[j] = expertRewards[j].add(bounty.amount.mul(assertions[j].bid).div(ap.winnerPool));
                         } else {
                             expertRewards[j] = expertRewards[j].sub(assertions[j].bid);
                         }
                     }
                 }
-
-                delete ap;
             }
+            delete ap;
```

Comment:
I guess we don't need this `delete` now that it's reinitialized on each loop. It would save some gas (5000) to remove it.